js/models/conversations.js

```diff
@@ -38,6 +38,8 @@
             conversation: this
         });
 
+        this.set({ 'muted': this.getMutedState() });
```

Comment:
this now amounts to a no-op: `this.set({'muted': this.get('muted')})` so we can drop it.